docs/api-reference.md

```diff
@@ -16,7 +16,7 @@ The Connection Resolver is made up of several components:
 
 ### connection-builder
 
-Builds the ODBC ConnectString or JDBC Connection URL.
+Builds the ODBC ConnectString or JDBC Connection URL. For JDBC Connection URL only, it is required that the conection-builder contains only non-secure attributes such as : server, port, dbname.
```

Comment:
For a JDBC Connection URL only, we require that the that the connection-builder contains only non-secure attributes, such as server, port, and dbname.